Utils/utils.ts

```diff
@@ -0,0 +1,89 @@
+import { Browser, BrowserContext, Locator, Page } from "@playwright/test";
+import { operations } from "../enums/operations.enum";
+import { stateEnum } from "../enums/state.enum";
+
+export class utils
+{
+    private readonly page : Page
+
+    constructor(page : Page)
+    {
+        this.page = page;
+    }
+
+    async goto(url : string, nextSelector : string, operation : operations.visible | operations.waitFor = operations.waitFor, timeout? : number) : Promise<boolean>
```

Comment:
no need for the isVisible here as you always need to use the waitForSelector, in addition, wait for will receive a state of selector as an optional parameter